lib/linktypes.c

```diff
@@ -409,6 +413,7 @@ bool demote_packet(libtrace_packet_t *packet)
 			trace_clear_cache(packet);
 			break;
 		default:
+                        assert(0);
```

Comment:
This looks like debug that should be removed

Reply:
Fixed in 91bd90e3283b171fb1f5d27b2827850ee72b25bd.